src/diffusers/image_processor.py

```diff
@@ -30,34 +30,26 @@ class VaeImageProcessor(ConfigMixin):
 
     Args:
         do_resize (`bool`, *optional*, defaults to `True`):
-            Whether to downscale the image's (height, width) dimensions to multiples of `vae_scale_factor`. Can accept
```

Comment:
Think this and the following changes to the VAE image processor were probably accidental no? Could we maybe revert this? :-)

Reply:
Yes, sorry!
I was using a previous commit when started and did "make style" so I guess it changed a bit the previous VAE. I will definitely revert

Reply:
Done!